.devcontainer/devcontainer.json

```diff
@@ -6,5 +6,13 @@
   "features": {
     "ghcr.io/devcontainers/features/docker-in-docker": {}
   },
+  "mounts": [
+    // To enable kernel modules for devcontainer
+    "source=/lib/modules,target=/lib/modules,type=bind,readonly",
+    "source=/usr/lib/modules,target=/usr/lib/modules,type=bind,readonly"
```

Comment:
cilium also checks debug loadable module, i am not sure if we need it here? any thoughts?

Reply:
I don't think it's a strict requirement